### DISCUSSION TITLE
Some information about calling in VoiceAttack

### DISCUSSION THREAD

**Discussion by YenShihHung** [EDCD/EDDI#2677] [EDDI VoiceAttack Q&A] [answered]
Hello, I would like to ask if I want to obtain
Information on carrier-based aircraft/ground vehicles (number of types/number on hand/maximum number)
The amount of ammunition for the weapon
Are these obtained by variables in ((EDDI ship loadout))? Will these variables update themselves?
Also what should I query if I want to get the readiness of the NPC Crew?

**Comment by Tkael** [comment, marked as answer]
Hi there. I'm not sure that I fully understand the intended questions but I will do my best to answer. 
1) There are no journal events which provide data on the ships stored in your shipyard on your carrier. EDDI could perhaps track individual dock events but it would quickly get out of sync if you ever play without EDDI. Or if you are asking about stocked vehicles from carrier outfitting then as far as I'm aware that information is also not available from the journal. 
2) The amount of ammunition in the hopper for weapons is not available from the player journal. 
3) NPC crew status is not available from the player journal though changes in status are recorded. So as long as EDDI is running it is possible to capture changes in crew status.

**Reply by YenShihHung** [reply]
thanks for your reply
So I can understand that the data obtained by EDDI comes from player logs
The data in the event function is valid only when the call is made
Only the variables mentioned in the VoiceAttack Integration page are automatically updated
The star list and player navigation list in the System are also not in the player log, right?

**Reply by YenShihHung** [reply]
I also encountered a problem: why can’t I adjust the value of this variable?
{TXT:Ship tiny/small/medium/large/huge hardpoint {INT:~i} module}
Even if the forward drop is shortened, the result is incorrect.
{TXT:Ship tiny hardpoint {INT:~i} module}

**Reply by Tkael** [reply]
> thanks for your reply So I can understand that the data obtained by EDDI comes from player logs The data in the event function is valid only when the call is made Only the variables mentioned in the VoiceAttack Integration page are automatically updated The star list and player navigation list in the System are also not in the player log, right?

Correct they are not.

**Reply by Tkael** [reply]
> I also encountered a problem: why can’t I adjust the value of this variable? {TXT:Ship tiny/small/medium/large/huge hardpoint {INT:~i} module} Even if the forward drop is shortened, the result is incorrect. {TXT:Ship tiny hardpoint {INT:~i} module}

I don't understand the question?
